test-data/unit/check-namedtuple.test

```diff
@@ -1,7 +1,7 @@
 [case testNamedTupleUsedAsTuple]
 from collections import namedtuple
 
-X = namedtuple('X', ['x', 'y'])
+X = namedtuple('X', 'x y')
```

Comment:
My change ends up requiring `list` to exist in all of these tests. It was easier to just change the namedtuple call than to add `[builtins fixtures/list.pyi]` everywhere. I left a few with the list to make sure we test that code path.